Formula/juju.rb

```diff
@@ -22,6 +22,136 @@ class Juju < Formula
 
   depends_on "go" => :build
 
+  resource "bcrypt" do
+    url "https://files.pythonhosted.org/packages/d8/ba/21c475ead997ee21502d30f76fd93ad8d5858d19a3fad7cd153de698c4dd/bcrypt-3.2.0.tar.gz"
+    sha256 "5b93c1726e50a93a033c36e5ca7fdcd29a5c7395af50a6892f5d9e7c6cfbfb29"
+  end
+
```

Comment:
This isn't right.

Reply:
yeah, we certainly need to revisit the command for few formulae about this. 😅